installer/install-clojure-lsp.cmd

```diff
@@ -1,5 +1,5 @@
 @echo off
 
 setlocal
-set VERSION=2021.02.01-20.37.52
+set VERSION=%1:-2021.02.01-20.37.52%

```

Comment:
I have no idea if this works, I used this [tool](https://daniel-sc.github.io/bash-shell-to-bat-converter/) to convert a bash expression to a bat one haha